Realm/RLMRealm.h

```diff
@@ -500,6 +500,69 @@ typedef void (^RLMNotificationBlock)(RLMNotification notification, RLMRealm *rea
  */
 - (BOOL)transactionWithoutNotifying:(NSArray<RLMNotificationToken *> *)tokens block:(__attribute__((noescape)) void(^)(void))block error:(NSError **)error;
 
+#ifdef REALM_ASYNC_WRITES
+
+/**
+ AsyncHandle A handle for asynchronous transaction returned
+ by `beginAsyncWriteTransaction:` or `commitAsyncWriteTransaction:` calls.
+ Can be used to cancel the transaction.
+*/
+typedef unsigned AsyncHandle;
```

Comment:
Bad grammar/punctuation on doc string. I'm also not sure "handle" is idiomatic for Realm. I would consider `AsyncTransactionToken` or `AsyncTransactionId`.

Reply:
I’d say it’s `AsyncTransactionId` as it really holds nothing but a number of a transaction. Token is something we’re use to retain the object (`NotificationToken`)